arbeitszeit_web/www/presenters/change_user_email_address_presenter.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases import change_user_email_address
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import UrlIndex
+
+
+@dataclass
+class ViewModel:
+    redirect_url: str | None
+
+
+@dataclass
+class ChangeUserEmailAddressPresenter:
+    notifier: Notifier
+    url_index: UrlIndex
+    translator: Translator
+
+    def render_response(
+        self, response: change_user_email_address.Response
+    ) -> ViewModel:
+        if response.is_rejected:
+            self.notifier.display_warning(
+                self.translator.gettext(
+                    "Something went wrong. Perhaps the email address is not valid or already in use."
```

Comment:
I guess it would be ideal if the diagnosis were more specific.  Would the user be able to contact a system administrator to find out exactly what went wrong?  Also: will the request be rejected if the user's requested new e-mail address is the same as the old one?

Reply:
This would need some differentiation on the use case side (`arbeitszeit/use_cases/change_user_email_address.py`). I did not see this as a priority to be honest. If you do, just comment here and I will fix it in an upcoming PR.

So far we have not discussed, if, when and how users should be able to contact a system admin. A good point for one of the next meetings IMO. 

Yes, when the requested new email exists, the request will be rejected.